01_getting_started/hello_world.py

```diff
@@ -106,21 +103,20 @@ def main():
 # running code locally. That means no waiting for long image builds when you've just moved a comma,
 # no fiddling with container image pushes, and no context-switching to a web UI to inspect logs.
 #
-# ### You can map over more data
+# ### Map over more data
 #
 # Change the `map` range from `20` to some large number, like `1170`. You'll see
 # Modal create and run even more containers in parallel this time.
 #
 # And it'll happen lightning fast!
 #
-# ### You can run a more interesting function
+# ### Run something more interesting
 #
-# The function `f` is obviously silly and doesn't do much, but in its place
+# The function `f` is silly and doesn't do much, but in its place
 # imagine something that matters to you, like:
 #
-# * Running [language model inference](/docs/examples/vllm_mixtral) or [fine-tuning](/docs/examples/slack-finetune)
-# * Manipulating [audio](/docs/examples/discord-musicgen) or [images](stable_diffusion_xl_turbo)
+# * Deploying [language model inference](/docs/examples/vllm_mixtral) or [fine-tuning](/docs/examples/slack-finetune) workloads.
```

Comment:
I wouldn't say that fine-tuning is a "workload" I want to "deploy".

And without a broader discussion of deployment as distinct from function execution, "running inference" doesn't imply to me that it can't be deployed.

I can see an argument for including deployment in the Modal hello world example, but that's not what we're doing here. 

More broadly, I don't see what "workloads" does for us here.

Reply:
Will edit it again. Inference and fine-tuning are quite different kinds of workloads and I think it missed the mark